rpcs3/Emu/RSX/RSXThread.cpp

```diff
@@ -2557,7 +2557,7 @@ namespace rsx
 	void thread::get_zcull_stats(u32 type, vm::addr_t sink)
 	{
 		u32 value = 0;
-		if (!g_cfg.video.disable_zcull_queries)
+		if (!g_cfg.video.disable_zcull_queries && !g_cfg.video.disable_video_output)
```

Comment:
Switching this path on the fly is dangerous and likely to hang the application. Applications actually care about the order they receive their reports/timestamps in and some engines are known to assert because of it.